security/cmd/istio_ca/main.go

```diff
@@ -306,11 +303,15 @@ func runCA() {
 	stopCh := make(chan struct{})
 	if !opts.serverOnly {
 		log.Infof("Creating Kubernetes controller to write issued keys and certs into secret ...")
+
+		citadelNamespace := env.RegisterStringVar("CITADEL_NS", "",
```

Comment:
is using `CITADEL_` prefix commonly used for other env-variables?

Reply:
Agreed on the `ns` being more idiomatic, fixed.

Did some digging on what the best prefix for the envvar would be. From [here](https://github.com/istio/istio/blob/3ce3bf9e90218699d18c65434822509936054cbe/mixer/adapter/kubernetesenv/kubernetesenv.go#L392), seems like `POD_NAMESPACE` would work fine.